app/lambda/create_session/lambda_function.py

```diff
@@ -0,0 +1,65 @@
+import json
+import sys
+import time
+import uuid
+
+sys.path.append("../")
+import boto3
+from lambda_base import LambdaBase
+
+
+class CreateSessionLambda(LambdaBase):
+    def __init__(self): 
+        pass
+
+    def CurTimeToEpoch(self): 
+        return int(time.time())
+
+    def EpochToCurTime(self, epoch): #int
+        return time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(epoch))
+
+    def putItem(self, PatientId, HealthCareProfessionalId, SessionName, TimeStampStart):
+        client = boto3.resource('dynamodb')
+        table = client.Table("Sessions")
+        tempId = uuid.uuid4().hex 
+        epochTime = self.CurTimeToEpoch()
+        SessionId = "s-"+tempId+str(epochTime)
+        ComprehendS3Path = "c-"+tempId+str(epochTime)
+        TranscribeS3Path = "t-"+tempId+str(epochTime)
+        AudioS3Path = "a-"+tempId+str(epochTime)
+        table.put_item(Item= {'SessionId': SessionId,
+                              'PatientId': PatientId,
+                              'HealthCareProfessionalId': HealthCareProfessionalId,
+                              'SessionName': SessionName,
+                              'ComprehendS3Path': ComprehendS3Path,
+                              'TranscribeS3Path': TranscribeS3Path,
+                              'AudioS3Path': AudioS3Path,
+                              'TimeStampStart': TimeStampStart,
+                              'TimeStampEnd': epochTime})
+        s3_client = boto3.resource('s3')
+        return SessionId
+
+    def handle(self, event, context):
+        print("event: {}".format(event))
+        try:
```

Comment:
Do you have future plans on adding in input validation?

If the patient id or healthcare professional id is None, we should return an invalid input error.

Learn more about errors and response codes [here](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html)
I would recommend for now just go over status codes : 200,400,404,500

Reply:
Will update the code in next pull request, when testing are uploaded.